src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -267,6 +267,7 @@ def fetch_codecov_data(
                 ref=ref if ref else branch,
                 ref_type="sha" if ref else "branch",
                 path=path,
+                has_error_commit=has_error_commit,
```

Comment:
I decided to pass this boolean as a parameter so we can set all the codecov tags together